utils/args/GenericfMRISurfaceProcessingPipeline.py

```diff
@@ -0,0 +1,62 @@
+"""
+Builds, validates, and excecutes parameters for the HCP script 
+/opt/HCP-Pipelines/fMRISurface/GenericfMRISurfaceProcessingPipeline.sh
+part of the hcp-func gear
+"""
+import os, os.path as op
+from .common import build_command_list, exec_command
+import logging
+
+log = logging.getLogger(__name__)
+
+def build(context):
+# if [[ -z "${FW_CONFIG_RegName}" ]] || [[ $(toupper "${FW_CONFIG_RegName}") = "EMPTY" ]]; then
+#   RegName="${hcpstruct_RegName}"
+# else
+#   RegName="${FW_CONFIG_RegName}"
+# fi
```

Comment:
I like having the command line version for reference of the conversion to python but I think it would make sense to put these comments next to the counterpart python code below?

Reply:
You found my `cruft`!